src/components/TextInput.jsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+
+export function TextInput(props) {
+  const { name, placeholder } = props;
+  const [value, setValue] = useState("");
+
+  return (
+    <input
+      type="text"
+      name={name}
+      placeholder={placeholder}
+      value={value}
+      onChange={(e) => setValue(e.target.value)}
+    ></input>
+  );
+}
```

Comment:
Waarom werkt dit zonder export default TextInput?

Reply:
export staat boven in voor de function. Vind dat altijd wat makkelijker, maar is persoonlijk denk ik.

export function TextInput(props) {

}